viewer/lib/entity/Entity.js

```diff
@@ -200,12 +200,91 @@ function getMesh (texture, jsonModel) {
   return mesh
 }
 
+function getPlayerMesh (texture, jsonModel) {
+  const bones = {}
+
+  const geoData = {
+    positions: [],
+    normals: [],
+    uvs: [],
+    indices: [],
+    skinIndices: [],
+    skinWeights: []
+  }
+  let i = 0
+  jsonModel.default.bones.forEach(async jsonBone => {
```

Comment:
Why the async lambda function?

Reply:
I am used to async cause I use await a lot and always forget to add the async, but I see there is no use here, let me quickly remove it here...